osu.Game/Tests/Visual/OnlinePlay/TestRoomRequestsHandler.cs

```diff
@@ -128,8 +128,12 @@ public bool HandleRequest(APIRequest request, APIUser localUser, OsuGameBase gam
         public void AddServerSideRoom(Room room)
         {
             room.RoomID.Value ??= currentRoomId++;
+
             for (int i = 0; i < room.Playlist.Count; i++)
+            {
                 room.Playlist[i].ID = currentPlaylistItemId++;
+                room.Playlist[i].OwnerID = room.Host.Value.OnlineID;
```

Comment:
As evidenced by the test failures, it appears some tests create rooms without a host.